aten/src/ATen/native/cpu/UnaryOpsKernel.h

```diff
@@ -28,23 +28,14 @@ extern DispatchStub<unary_fn> sqrtImpl;
 extern DispatchStub<unary_fn> tanhImpl;
 extern DispatchStub<unary_fn> truncImpl;
 
+extern DispatchStub<unary_fn> cosImpl;
+extern DispatchStub<unary_fn> coshImpl;
+extern DispatchStub<unary_fn> sinImpl;
+extern DispatchStub<unary_fn> sinhImpl;
+extern DispatchStub<unary_fn> tanImpl;
 
 // Missing unary functions
 // digamma
 // lgamma
 
-// TODO: See below
-// erfinv
-// fill
-// frac
-// clone
-// contiguous
-// clamp/_min/_max
-// neg
-// reciprocal
-// sigmoid
-// sign
-// zero
```

Comment:
A few of those are still not implemented I think

Reply:
I'm adding that later today/tomorrow.